dev_tools/scripts/db_integrity_test.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# Insert some test data in the db
+docker exec db psql -U postgres -d test -c "INSERT INTO local_ega.main(submission_file_path,submission_user,submission_file_extension,status,encryption_method) VALUES('test.c4gh','dummy','c4gh','INIT','CRYPT4GH');"
+
+# Build, run and execute the pg_basebackup action
+docker container run --rm -i --name pg-backup --network=host $(docker build --build-arg USER_ID=$(id -u) -f dev_tools/Dockerfile-backup -q -t backup .) /bin/sda-backup --action pg_basebackup
+
+# Find the name of the copy in the S3 and check the length
+DBCOPY=$(s3cmd -c dev_tools/s3conf ls s3://dumps/ | grep ".tar" | cut -d '/' -f4)
+DBCOPYLENGTH=`echo -n "$DBCOPY" | wc -m`
+
+# Find the name of the backup image created in the previous step and check the length
+IMAGENAME=$(docker images -q backup)
+NAMELENGTH=`echo -n "$IMAGENAME" | wc -m`
+
+# Make sure that the image and the db copy exist before moving on
+if [ $NAMELENGTH = 0 ] || [ $DBCOPYLENGTH = 0 ]; then
+    echo "the image or the db copy is missing"
+    exit 1
+# Get the db copy from S3
+else
+    docker container run -v $(pwd)/tmp:/home --rm -i --name pg-backup --network=host backup /bin/sda-backup --action pg_bb-restore --name "$DBCOPY"
+fi
```

Comment:
```suggestion
# Get the db copy from S3
fi

docker container run -v $(pwd)/tmp:/home --rm -i --name pg-backup --network=host backup /bin/sda-backup --action pg_bb-restore --name "$DBCOPY"

```
No need for the ELSE statement, if the check hold true the script will terminate before it gets to the ELSE part.

Reply:
Correct, it is updated now